lib/config.py

```diff
@@ -24,6 +24,74 @@
 from lib import utils
 
 LOG = logging.getLogger(__name__)
+BUILD_REPO_ARGS = {
+    '--build-versions-repository-url':
+        dict(help='Build versions repository URL'),
+    '--build-version':
+        dict(help='Select build version from versions repository'),
+    '--build-versions-repo-dir':
+        dict(help='Directory to clone the build versions repository',
+             default='./components'),
+    '--http-proxy':
+        dict(help='HTTP proxy URL'),
+}
+PACKAGE_ARGS = {
+    '--packages -p':
```

Comment:
I'd prefer to use a tuple here. This way, we'll not need to split the string afterwards.
Tuples are immutable and hence can be used as dict keys.
`('--packages', '-p'): dict(...)`